README.md

```diff
@@ -293,6 +309,30 @@ if (Meteor.isServer) {
 
 Other table settings should work normally, except that all fields will be sorted by value, even if using `fn`. The fields setting is required when using a server-side collection.
 
+### Settings
+
+Possible options:
+- autoQuoteSearchterm (Boolean - *default=* **true**):
```

Comment:
actually, "enableRegex" might be better - it's more clear why you'd use this option